quiz.js

```diff
@@ -1,9 +1,9 @@
 
-var button = document.querySelector('#form input[type=submit]');
+// var button = document.querySelector('#form input[type=submit]');
 var lines;
 var spaces;
 var spaceCount = 0;
-var enterKey = document.querySelector('#button');
+var enterKey = document.getElementById('button');
```

Comment:
So, this button should undoubtedly run your program and you should have an event listeners on it. However, you should also have event listeners on each input box that listens for a certain keypress.